### PR TITLE
Update whats-new-in-defender-for-office-365.md

### PR DESCRIPTION
**Pull request by VipulMSFT**
Adding the reference for URLClickEvents in What's new section.

### CODE REVIEW

File: microsoft-365/security/office-365-security/whats-new-in-defender-for-office-365.md [hunk 1]
```diff
@@ -46,6 +46,10 @@ For more information on what's new with other Microsoft Defender security produc
 - [What's new in Microsoft Defender for Identity](/defender-for-identity/whats-new)
 - [What's new in Microsoft Cloud App Security](/cloud-app-security/release-notes)
 
+## April 2022
+
+- [Introducing the URLClickEvents table in Microsoft 365 Defender Advanced Hunting](https://techcommunity.microsoft.com/t5/microsoft-defender-for-office/introducing-the-urlclickevents-table-in-advanced-hunting-with/ba-p/3295096): Introducing the UrlClickEvents table in advanced hunting with Microsoft Defender for Office 365
```
Comment (JohanFreelancer9) [inline microsoft-365/security/office-365-security/whats-new-in-defender-for-office-365.md]:
```suggestion
- [Introducing the URLClickEvents table in Microsoft 365 Defender Advanced Hunting](https://techcommunity.microsoft.com/t5/microsoft-defender-for-office/introducing-the-urlclickevents-table-in-advanced-hunting-with/ba-p/3295096): Introducing the UrlClickEvents table in advanced hunting with Microsoft Defender for Office 365.
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review - @VipulMSFT, kindly commit the minor suggested change. Many thanks.

**Comment by VipulMSFT** [comment]
@JohanFreelancer9 - committed the changes. Thanks for the review.

**Review by JohanFreelancer9 — APPROVED** [review]

**Comment by VipulMSFT** [comment]
@MSFTTracyP @JohanFreelancer9 can you please merge this PR

**Comment by VipulMSFT** [comment]
Hi @MSFTTracyP @JohanFreelancer9, can you please merge this PR

**Comment by JohanFreelancer9** [comment]
@VipulMSFT - unfortunately, I do not have the rights to merge. @yogkumgit?

**Comment by yogkumgit** [comment]
@tiburd Please us here. Thanks

**Comment by tiburd** [comment]
@VipulMSFT 
Hi,

Thank you for your submission.

Please note:
- The file conflict must be resolved before the PR can be merged.
- At your earliest convenience, please complete your account setup so that your GitHub and Microsoft accounts are connected:
   - [Set up your accounts](https://review.docs.microsoft.com/en-us/office-authoring-guide/contribute-get-started-setup-github-ro?branch=main)
- This pull request was opened in the public repo. PMs and content developers should work in the private repo, as described in [the authoring guide](https://review.docs.microsoft.com/en-us/Office-Authoring-Guide/office-docs-authoring-overview?branch=main). We encourage you to use the private repo for future contributions to Microsoft Docs.

Thanks!

**Pull request merged**